src/Rector/AbstractRector.php

```diff
@@ -435,4 +442,18 @@ private function connectParentNodes(Node $node): void
         $nodeTraverser->addVisitor(new ParentConnectingVisitor());
         $nodeTraverser->traverse([$node]);
     }
+
+    private function printDebugCurrentFileAndRule(): void
+    {
+        if (! $this->rectorOutputStyle->isDebug()) {
+            return;
+        }
+
+        $this->rectorOutputStyle->writeln('[file] ' . $this->file->getRelativeFilePath());
+
+        // prevent spamming with the same class over and over
```

Comment:
this comment is invalid, as it recorded at : 

```php
        if ($this->previousAppliedClass === static::class) {
            return;
        }
```

ref https://github.com/rectorphp/rector-src/pull/1933/files#diff-1926ef5eec07704744c3f56345a1755c5863d2672344add93d58f81c21c2b24f